Casks/s/soundsource@test.rb

```diff
@@ -1,21 +1,23 @@
 cask "soundsource@test" do
-  version "5.7.6,9,20250204,0936"
-  sha256 "7fade114292b99708fd4425c57d13f6be55b3bd73619c0dd45a924ddc94f3efe"
+  version "5.7.6,10,20250206,1907,A"
+  sha256 "4fcd5a7b858b9fd590a0d96dfbe0f18a5408fab1106948058e2fb6d1f2cc8bb9"
 
-  url "https://download.rogueamoeba.com/builds/SoundSource/SoundSource_#{version.csv.first.no_dots}700#{version.csv.second}_#{version.csv.third}_#{version.csv.fourth}.zip"
+  url "https://download.rogueamoeba.com/builds/SoundSource/SoundSource_#{version.csv.first.no_dots}700#{version.csv.fifth}_#{version.csv.third}_#{version.csv.fourth}.zip"
   name "SoundSource"
   desc "Sound and audio controller"
   homepage "https://rogueamoeba.com/soundsource/"
 
   # The livecheck uses a hard-coded system version number in the url that corresponds to the latest macOS version
   livecheck do
-    url "https://rogueamoeba.net/ping/versionCheck.cgi?format=sparkle&system=150&wantsTestReleases=true&bundleid=com.rogueamoeba.soundsource&platform=osx&version=#{version.csv.first.no_dots}700#{version.csv.second}"
-    regex(/SoundSource[._-]v?(?:\d+)[._-](\d+)[._-](\d+)\.zip/i)
+    url "https://rogueamoeba.net/ping/versionCheck.cgi?format=sparkle&system=150&wantsTestReleases=true&bundleid=com.rogueamoeba.soundsource&platform=osx&version=#{version.csv.first.no_dots}700#{version.csv.fifth || version.csv.second}"
+    regex(/SoundSource[._-]v?(?:\d+)([A-z]+)?[._-](\d+)[._-](\d+)\.zip/i)
     strategy :sparkle do |item, regex|
       match = item.url&.match(regex)
       next if match.blank?
 
-      "#{item.version.sub("fc", ",")},#{match[1]},#{match[2]}"
+      return "#{item.version.sub("fc", ",")},#{match[1]},#{match[2]}" if match.captures.length <= 2
+
+      "#{item.version.sub("fc", ",")},#{match[2]},#{match[3]},#{match[1]}"
```

Comment:
```suggestion
      "#{item.version.sub("fc", ",")},#{match[2]}#{"," + match[3] if
match[3]},#{match[1]}"
```

We can avoid duplicating this line.